src/repositories/metadataRepository.ts

```diff
@@ -0,0 +1,41 @@
+import { Db } from 'mongodb';
+import { IConfig } from 'config';
+import { BaseRepository } from './baseRepository';
+import { ILogger } from '../services/logger';
+
+/**
+ * Manages metadata for parser builds.
+ */
+export class MetadataRepository extends BaseRepository {
+  constructor(db: Db, config: IConfig, logger: ILogger) {
+    super(config, logger, 'MetadataRepository', db.collection('metadata'));
```

Comment:
The collection name is sometimes passed down as an env variable passed down from param store for other existing repositories. But I'm not sure what its benefit is when the collection name is expected to be static for every environment.

I figured keeping it hardcoded here would reduce unnecessary envs and the name itself isn't sensitive, but let me know if the preference is to make env variables

Reply:
I think this is totally fine to use over environment variables. I think it would also be worthwhile to assign the collection name to a `const` variable if that collection name is used elsewhere, just to prevent the possibility of a typo in one place and not another.